applications/FluidDynamicsApplication/custom_elements/alternative_qs_vms_dem_coupled.h

```diff
@@ -225,11 +225,16 @@ class AlternativeQSVMSDEMCoupled : public QSVMS<TElementData>
     ///@name Protected member Variables
     ///@{
 
-    int mInterpolationOrder = 1;
-    DenseVector <BoundedMatrix<double,Dim,Dim>> mViscousResistanceTensor;
+
     // Velocity subscale history, stored at integration points
     DenseVector< array_1d<double,Dim> > mPredictedSubscaleVelocity;
     DenseVector< array_1d<double,Dim> > mPreviousVelocity;
+    DenseVector <BoundedMatrix<double,Dim,Dim>> mViscousResistanceTensor;
+    int mInterpolationOrder = 1;
```

Comment:
Not sure what this `mInterpolationOrder` is, but if it indicates the order of the geometry interpolation, you can make it a `static constexpr` value from the template parameters to know it at compile time and avoiding the storage at each element (take it as a suggestion for the future, not as a blocker)